test/fw/ptl/utils/plugins/ptl_test_runner.py

```diff
@@ -744,10 +749,221 @@ def __are_requirements_matching(param_dic=None, test=None):
             if not eff_tc_req['no_comm_on_mom']:
                 return False
 
+    def hardware_report(self):
+        """
+        function used to get hardware report
+        in every 5 minutes
+        """
+        du = DshUtils()
+        self.hardware_report_timer = Timer(300, self.hardware_report)
+        self.hardware_report_timer.start()
+        self.timer_start = True
+        # mom hardware report
+        for hostname in self.param_dict['moms']:
```

Comment:
also why only moms? shouldn't we do for servers, scheds, and comms also?

Reply:
I also did for sever. is scheduler also a different machine.

Reply:
I don't see use of self.param_dict['servers'] in this method? can you please tell me line no. where you used it?

Reply:
I have added servers and comms block.